sheerwater_benchmarking/baselines/ecmwf.py

```diff
@@ -413,17 +421,23 @@ def ecmwf_agg(start_time, end_time, variable, forecast_type,
             - None: no mask
         verbose (bool): Whether to print verbose output.
     """
+    lons, lats, _ = get_grid(grid)
+    global_grid = get_global_grid(grid)
```

Comment:
I want to make ECMWF_rolled / ERA5_rolled only global functions, and the regional grids get applied at the final step. I'd love some feedback on the API here. I could also restrict the potential grids input into this function (this just gets the corresponding global grid, not matter what you pass).